src/components/help/help.component.js

```diff
@@ -80,13 +83,14 @@ const Help = ({
             rel: "noopener noreferrer",
           }
         : {
-            role: "tooltip",
+            role: "button",
```

Comment:
Should this be a `button`? A screen reader will read out that the element is clickable that's all. On a Mac it'll read out something like `You are currently on a button, To click this button press control-option-space` when Help is focused.

Reply:
yes, this definitely should be a button, otherwise it will state it is a group. Tooltip role was added incorrectly, as it should be set on a popup.

Reply:
That's fine. If that is what it is meant to be, I'm good with that 😄